config/defaults.conf

```diff
@@ -342,6 +342,8 @@ pair_attn = 1  // If true, use attn in sentence-pair classification/regression t
 d_hid_attn = 512  // Post-attention LSTM state size.
 shared_pair_attn = 0  // If true, share pair_attn parameters across all tasks that use it.
 d_proj = 512  // Size of task-specific linear projection applied before before pooling.
+pool_type = "max" // Type of pooling to reduce sequences of vectors into a single vector.
+                  // Options: "max", "mean", "first", "final"
```

Comment:
Idea, though not really a suggestion: Have an 'auto' option, maybe yielded by a blank string here, that uses max for most encoders, first for BERT, and final for GPT.

Reply:
That's nice to have but in the interest of getting this through, I'm going to skip for now.